test/application_system_test_case.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
```

Comment:
- [ ] Kindly add unit and integration tests to your App as the requirements say

![image](https://user-images.githubusercontent.com/65068771/190843002-cdd135a0-bb24-468d-ad39-01da3616db8b.png)